aws_lambda_builders/workflows/custom_make/actions.py

```diff
@@ -46,6 +61,7 @@ def __init__(self, artifacts_dir, scratch_dir, manifest_path, osutils, subproces
         self.osutils = osutils
         self.subprocess_make = subprocess_make
         self.build_logical_id = build_logical_id
+        self.working_directory = working_directory if working_directory else scratch_dir
```

Comment:
Why this implied behavior? we should document that in the docstring as well.

Reply:
This is the current default behaviour. Custom builder run against the scratch_dir, so if there is no working_directory passed by the customers, we should use the current default behaviour.  I updated the docstring.